db/migrate/20240205171918_create_greetings.rb

```diff
@@ -0,0 +1,8 @@
+class CreateGreetings < ActiveRecord::Migration[7.0]
+  def change
+    create_table :greetings do |t|
+
+      t.timestamps
+    end
```

Comment:
- [ ] There's no column in the `greetings` table to save the greetings in the database. Please make sure to add a column to the table.

Reply:
Dear @thecodechaser, 
Thanks for the suggestion, I appreciate it, 
I solved it .
Best wishes,